examples/tf/unsupervised_advanced_tf.py

```diff
@@ -105,7 +105,9 @@ def myregularizer(regularized_params):
 final_distribution = outcomes['outputs']
 
 # Use a helper function to sample fock states from this state.
-# They should show a similar distribution to the training data
+# They should show a similar distribution to the training data.
+# If the output of the function is a tensor, 
+# you can use sample_from_distribution from qmlt.tf.helpers
```

Comment:
Would be great if you could shorten the description. Lines 109 & 110 seem redundant to me?